### PR TITLE
Move raft timeout parameters into a storage.StoreConfig struct.

### PR DESCRIPTION
**Pull request by bdarnell**
Increase the defaults to something a little more sane for non-test use.

### CODE REVIEW

File: kv/local_sender_test.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func TestLocalSenderAddStore(t *testing.T) {
 	}
 }
 
-func TesLocalSendertGetStoreCount(t *testing.T) {
+func TestLocalSendertGetStoreCount(t *testing.T) {
```
Comment (petermattis) [inline kv/local_sender_test.go]: `s/Sendert/Sender/g`

Comment (bdarnell) [inline kv/local_sender_test.go]: Ah, that's where the missing t went. Fixed.

**Comment by petermattis** [comment]
LGTM

File: server/server.go [hunk 2]
```diff
@@ -115,7 +115,8 @@ func NewServer(ctx *Context) (*Server, error) {
 
 	s.kvDB = kv.NewDBServer(sender)
 	s.kvREST = kv.NewRESTServer(s.kv)
-	s.node = NewNode(s.kv, s.gossip)
+	// TODO(bdarnell): make StoreConfig configurable.
```
Comment (spencerkimball) [inline server/server.go]: Do you have anything in mind for this? I'd just remove the TODO unless there's a good reason we're going to want to change default values in production (if yes, then the StoreConfig should perhaps be part of the server.Context).

Comment (bdarnell) [inline server/server.go]: We'll definitely want to change this in production - single-datacenter deployments will want much smaller election timeouts than geographically-distributed ones.

**Pull request merged**